ワークフロー「拡張機能パッケージの公開」がエラー終了

現在ワークフロー"release_extension_packages.yml"を実行すると、
以下の箇所でエラーが発生している。

`Run npm run build
  npm run build
  shell: /usr/bin/bash -e {0}
  env:
    FORCE_JAVASCRIPT_ACTIONS_TO_NODE24: true
    pythonLocation: /opt/hostedtoolcache/Python/3.14.4/x64
    PKG_CONFIG_PATH: /opt/hostedtoolcache/Python/3.14.4/x64/lib/pkgconfig
    Python_ROOT_DIR: /opt/hostedtoolcache/Python/3.14.4/x64
    Python2_ROOT_DIR: /opt/hostedtoolcache/Python/3.14.4/x64
    Python3_ROOT_DIR: /opt/hostedtoolcache/Python/3.14.4/x64
    LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.14.4/x64/lib

> app@1.0.3 build
> python3 scripts/generate_png_icons.py && python3 scripts/generate_animation_registry.py && python3 scripts/check_version.py && python3 scripts/create_package.py

Error: No module named 'playwright'. Please install it to generate extension icons.
Error: Process completed with exit code 1.`

--- a/.github/workflows/release_extension_packages.yml
+++ b/.github/workflows/release_extension_packages.yml
@@ -42,6 +42,26 @@ jobs:
       - name: Install dependencies
         run: npm ci
 
+      - name: Cache Playwright browsers
+        uses: actions/cache@v4
+        id: playwright-cache
+        with:
+          path: ~/.cache/ms-playwright
+          key: ${{ runner.os }}-playwright-${{ hashFiles('package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-playwright-
+
+      - name: Install Playwright Browsers
+        if: steps.playwright-cache.outputs.cache-hit != 'true'
+        run: npx playwright install chromium --with-deps
+
+      - name: Install Playwright Dependencies (if cache hit)
+        if: steps.playwright-cache.outputs.cache-hit == 'true'
+        run: npx playwright install-deps chromium
+
+      - name: Install Python Playwright
+        run: pip install playwright
+
       - name: Build packages
         run: npm run build
 

